ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>What is the purpose of this cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>